File: android-app/app/src/main/java/com/findle/ruaiunlocker/data/model/HostsInfo.kt
package com.findle.ruaiunlocker.data.model

data class HostsInfo(
    val lastUpdate: String,
    val entriesCount: Int,
    val fileSize: Long,
    val entries: List<HostEntry> = emptyList()
)

data class HostEntry(
    val ip: String,
    val domain: String,
    val section: String = ""
)

File: windows/Models/HostsInfo.cs
using System.Collections.Generic;

namespace RUAIUnlocker.Models
{
    public class HostsInfo
    {
        public string LastUpdate { get; set; }
        public int EntriesCount { get; set; }
        public long FileSize { get; set; }
        public List<HostEntry> Entries { get; set; }

        public HostsInfo()
        {
            LastUpdate = string.Empty;
            EntriesCount = 0;
            FileSize = 0;
            Entries = new List<HostEntry>();
        }
    }
}
